aws/resource_aws_ecs_service_test.go

```diff
@@ -581,6 +599,86 @@ resource "aws_ecs_service" "mongo" {
 `, rInt, rInt)
 }
 
+func testAccAWSEcsServiceWithLaunchTypeFargate(rInt int) string {
+	return fmt.Sprintf(`
+data "aws_availability_zones" "available" {}
+
+resource "aws_vpc" "main" {
+	cidr_block = "10.10.0.0/16"
+}
+
+resource "aws_subnet" "main" {
+	count = 2
+	cidr_block = "${cidrsubnet(aws_vpc.main.cidr_block, 8, count.index)}"
+	availability_zone = "${data.aws_availability_zones.available.names[count.index]}"
+	vpc_id = "${aws_vpc.main.id}"
+}
+
+resource "aws_security_group" "allow_all_a" {
+	name        = "allow_all_a"
```

Comment:
Can you add some randomisation to the names used by tests? this avoids issues when running them in parallel.